src/libraries/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs

```diff
@@ -1660,11 +1645,11 @@ public bool IsEmpty
         /// elements.</exception>
         /// <exception cref="ArgumentException">
         /// <paramref name="key"/> is of a type that is not assignable to the key type <typeparamref
-        /// name="TKey"/> of the <see cref="Dictionary{TKey,TValue}"/>. -or-
+        /// name="TKey"/> of the <see cref="ConcurrentDictionary{TKey,TValue}"/>. -or-
```

Comment:
Shouldn't this just be `<inheritdoc />`?

Reply:
My understanding is that It could be if `IDictionary.Add` was providing xml docs. But it does not:

https://github.com/dotnet/runtime/blob/a9cdcd5716424c557d7a135d65b7ab88803de1dd/src/libraries/System.Private.CoreLib/src/System/Collections/IDictionary.cs#L30-L31

cc @carlossanlop